set.mm

```diff
@@ -632273,6 +632273,90 @@ fixed reference functional determined by this vector (corresponding to
       TUICVSAUJUTUKPUQVQVTVRVALPUQVLWAVNVLDKCWAULADUMUNPUTVAURUOUPS $.
   $}
 
+
+  ${
+    $d K k $.  $d M k $.  $d N k $.  $d k ph $.
+    prodsplit.1 $e |- ( ph -> M e. ZZ ) $.
+    prodsplit.2 $e |- ( ph -> N e. ZZ ) $.
+    prodsplit.3 $e |- ( ph -> M <_ N ) $.
+    prodsplit.4 $e |- ( ph -> K e. NN0 ) $.
+    prodsplit.5 $e |- ( ( ph /\ k e. ( M ... ( N + K ) ) ) -> A e. CC ) $.
+    $( Product split into two factors, original by icecream.  (Contributed by
```

Comment:
Steve Nguyen does not appear as icecream in set.mm.  Use SN to refer to him.

Reply:
I've used his full name for consistency instead of SN, I hope that is ok.